lib/get-info.js

```diff
@@ -827,54 +828,29 @@ var Getters = {
             }
 
             try {
+                var $ = cheerio.load(res, { xmlMode: true });
+                switch ($("gphoto\\:videostatus").text()) {
+                    case "final":
+                        break; /* Incoming Fun. */
+                    case "pending":
+                        return cb("The video is still being processed.", null);
```

Comment:
Can you add these errors to the switch case in `initGooglePlus` in mediarefresher.js so that they don't get logged in the global error log?